tools/osbuilder/node-builder/azure-linux/README.md

```diff
@@ -0,0 +1,285 @@
+  runtime_type = "io.containerd.runc.v1"
+[plugins."io.containerd.grpc.v1.cri".containerd.runtimes.katacli.options]
+  NoPivotRoot = false
+  NoNewKeyring = false
+  ShimCgroup = ""
+  IoUid = 0
+  IoGid = 0
+  BinaryName = "/usr/bin/kata-runtime"
+  Root = ""
+  CriuPath = ""
+  SystemdCgroup = false
+[proxy_plugins]
+  [proxy_plugins.tardev]
+    type = "snapshot"
+    address = "/run/containerd/tardev-snapshotter.sock"
+[plugins."io.containerd.grpc.v1.cri".containerd.runtimes.kata-cc]
+  snapshotter = "tardev"
+  runtime_type = "io.containerd.kata-cc.v2"
+  privileged_without_host_devices = true
+  pod_annotations = ["io.katacontainers.*"]
+  [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.kata-cc.options]
+    ConfigPath = "/opt/confidential-containers/share/defaults/kata-containers/configuration-clh-snp.toml"
+```
+
+Restart containerd:
+
+```sudo systemctl restart containerd```
+
+# Install general build dependencies
+
+```sudo dnf install -y git vim golang rust build-essential protobuf-compiler protobuf-devel expect openssl-devel clang-devel libseccomp-devel parted qemu-img btrfs-progs-devel device-mapper-devel cmake fuse-devel jq curl kata-packages-uvm-build```
+
+If you intend to build the confpods UVM, install the following package:
+```sudo dnf install -y kernel-uvm-devel``
+
+## Optional: Build and deploy the containerd fork from scratch
+
+```
+git clone --depth 1 --branch tardev-v1.7.7 https://github.com/microsoft/confidential-containers-containerd.git
+pushd confidential-containers-containerd/
+GODEBUG=1 make
+popd
+```
+
+Overwrite existing containerd binary, restart service:
+```
+sudo cp -a -S .bak -b confidential-containers-containerd/bin/containerd /usr/bin/containerd
+sudo systemctl restart containerd
+```
+
+# Use Kata tooling to build the Kata host and guest components
+
+If these instructions are not being followed based on having cloned its hosting repository onto the system, clone the kata-containers repository:
+```git clone --depth 1 --branch mahuber/reproducible-builds https://github.com/microsoft/kata-containers.git```
+
+To build Azure Linux's kata-containers package components and UVM, run:
+```
+pushd kata-containers/tools/osbuilder/node-builder/azure-linux
+make all
+popd
+```
+To build the kata-containers-cc package components and UVM, run `make all-confpods`.
+The `all` target runs the `clean[-confpods]`, `package[-confpods]` and `uvm[-confpods]` targets in a single step (the `uvm` target depends on the `package` target).
+
+**Note:** By default, a UVM with Kata Agent enforcing a restrictive policy will be built, this requiring to generate valid pod security policy annotations using the `genpolicy` tool. While always recommended to do keep doing so, a UVM with a permissive security policy can be built by adapting the file `tools/osbuilder/node-builder/azure-linux/uvm_build.sh` and changing the `AGENT_POLICY_FILE` variable assignment to `allow-all.rego`
+
+# Install the built components
```

Comment:
The output of "sudo make deploy-confpods" is too cryptic for me - e.g., I couldn't figure out easily if it installs utarfs.

Reply:
Addressed in next iteration, as discussed.